get_coverage_for_challenge.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+set -x
+set -e
+set -u
+set -o pipefail
+
+SCRIPT_CURRENT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+CHALLENGE_ID=$1
+RUBY_TEST_REPORT_CSV_FILE="${SCRIPT_CURRENT_DIR}/coverage/results.csv"
+RUBY_CODE_COVERAGE_INFO="${SCRIPT_CURRENT_DIR}/coverage.tdl"
+
+( cd ${SCRIPT_CURRENT_DIR} && \
+    $HOME/.rbenv/shims/bundle install && \
+    $HOME/.rbenv/shims/rake test || true 1>&2 )
```

Comment:
Is the coverage generated per line or per package?

Reply:
Good point, it might be per file, I will check this out.

Reply:
I think we need to improve our container test by adding a test case where we expect the coverage to be partial. We need to think about that.

Reply:
Yes when we are done with these, we need to upgrade our tests to catch for other languages

Reply:
Done for ruby! Multiple files are handled now!